src/diffusers/__init__.py

```diff
@@ -15,6 +15,7 @@
     is_torchsde_available,
     is_transformers_available,
 )
+from .utils.testing_utils import DIFFUSERS_SLOW_IMPORT
```

Comment:
@patrickvonplaten I think this line would slow down the imports if torch is installed, since torch would get imported from testing utils even if slow import was set to False. 

https://github.com/huggingface/diffusers/blob/24fda526724e55c818577916bc866598e83f1a1d/src/diffusers/utils/testing_utils.py#L58-L60

Reply:
Nice catch!